javascript/organizing-js/organizing-introduction.md

```diff
@@ -2,9 +2,9 @@
 
 One of the most daunting parts of JavaScript is learning how to organize your code. The reason this subject can be so overwhelming is _not_ because JavaScript is so much more complex than other languages, but because it is incredibly forgiving! Many languages force you into using specific patterns and data structures in your code but that is not true in JavaScript.
 
-In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something you can set that up in a couple lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care to organize your code and because JavaScript is such a flexible language how you do that is entirely up to you. For many coders making decisions about design patterns is crippling so we're here to help.
+In the beginning, this is a great thing! For example, if you just want to make a simple button on your webpage do something, you can set that up in a couple of lines of code. However, as your program becomes more complex, it can become hard to maintain unless you take care to organize your code and, because JavaScript is such a flexible language, how you do that is entirely up to you. For many coders making decisions about design patterns is crippling so we're here to help.
```

Comment:
This doesn't quite look like my suggestion, it just looks like you rolled the sentence back.

Reply:
I went ahead and made this edit, trying to knock out all the PRs